doc/source/manual/xtrabackup_sst.rst

```diff
@@ -218,8 +240,10 @@ in :file:`my.cnf` before you enable this option.
 
 .. option:: cpat
 
-Used to define the files that need to be deleted in the :term:`datadir`
-before running SST,
+    :Default: ``'.*\.pem$\|.*init\.ok$\|.*galera\.cache$\|.*sst_in_progress$\|.*\.sst$\|.*gvwstate\.dat$\|.*grastate\.dat$\|.*\.err$\|.*\.log$\|.*RPM_UPGRADE_MARKER$\|.*RPM_UPGRADE_HISTORY$'``
+
+Used to define the files
+that need to be deleted in the :term:`datadir` before running SST,
```

Comment:
The description is incorrect.  This option is used to specify files that won't be deleted.  Everything else in the datadir will be deleted EXCEPT for these files.